src/components-examples/cdk/overlay/cdk-overlay-basic/cdk-overlay-basic-example.html

```diff
@@ -0,0 +1,27 @@
+<!-- This button triggers the overlay and is it's origin -->
+<button
+  mat-raised-button
+  (click)="isOpen = !isOpen"
+  type="button"
+  cdkOverlayOrigin
+  #trigger="cdkOverlayOrigin"
+>
+  {{isOpen ? "Close" : "Open"}}
+</button>
+
+<!-- This template displays the overlay content and is connected to the button -->
+<ng-template
+  cdkConnectedOverlay
+  [cdkConnectedOverlayOrigin]="trigger"
+  [cdkConnectedOverlayOpen]="isOpen"
+>
+  <mat-card>
```

Comment:
I general we try to avoid using Material components in the CDK examples, because the CDK shouldn't have any external dependencies, aside from the framework.

Reply:
Done